code/modules/mob/mob.dm

```diff
@@ -1347,6 +1347,9 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 
 /mob/proc/spin(spintime, speed)
 	set waitfor = 0
+	if(!spintime || !speed || spintime > 100)
```

Comment:
I'd just have this be:

```
/mob/proc/spin(spintime, speed)
	set waitfor = 0
	var/D = dir
	if((spintime < 1) || (speed < 1) || !spintime || !speed)
		return
```

And fail silently. This doens't really need a stack trace.

Reply:
I would keep the stack trace still, since this is going to be called by admins who don't know the proc properly, so user feedback would be good here.